### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by PokhodenkoSA**

### CODE REVIEW

**Comment by PokhodenkoSA** [comment]
`scripts\build_for_conda.sh` is not actual now. Should I remove it?

File: README.md [hunk 1]
```diff
@@ -3,21 +3,40 @@ What?
...
+1. Create and activate conda build environment
+```bash
+conda create -n build-env conda-build
+conda activate build-env
+```
+2. Build conda package
+```bash
+conda build conda-recipe
+```
+On Windows to cope with long file names:
```
Comment (diptorupd) [inline README.md:26]: (fatal error C1083: Cannot open compiler generated file)

File: README.md [hunk 2]
```diff
@@ -3,21 +3,40 @@ What?
...
+========
+See examples in folder `examples`.
 
-Examples:
-===
-   Run create_sycl_queues.py under examples.
-
-   `python examples/create_sycl_queues.py`
+Run examples:
+```bash
```
Comment (diptorupd) [inline README.md:40]: Mention we need to source call "%ONEAPI_ROOT%compiler\latest\env\vars.bat"

File: README.md (hunk 1 above)
Comment (PokhodenkoSA) [inline README.md:26]:
```suggestion
On Windows to cope with [long file names](https://github.com/IntelPython/pydppl/issues/15):
```

File: README.md (hunk 2 above)
Comment (PokhodenkoSA) [reply README.md:52]: See `Using PyDPPL`

**Pull request merged**